pages/ActivityStreams/Overview.md

```diff
@@ -6,7 +6,7 @@ menu: Activity Streams
 
 # Activity Streams Overview
 
-Content shared via the DSNP will generally consist of URIs pointing to documents containing Activity Streams JSON objects.
+Content shared via the DSNP will generally consist of URLs pointing to documents containing Activity Streams JSON objects.
```

Comment:
We need to get standard on URI vs URL or what we mean by each...

Reply:
I changed this at @acruikshank's request: https://github.com/LibertyDSNP/spec/pull/90#discussion_r657971374

should I change it back?